ci-operator/jobs/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-4.7-postsubmits.yaml

```diff
@@ -9,7 +9,6 @@ postsubmits:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
-    max_concurrency: 1
     name: branch-ci-kubevirt-hyperconverged-cluster-operator-release-4.7-images
```

Comment:
Removing these lines looks suspicious.... What removed them? We added these for a reason.